libnymea-modbus/tools/README.md

```diff
@@ -179,12 +225,12 @@ Example block:
 Change into your plugin sub directory.
 Assuming you wrote the registers.json file you can run now following command to generate your modbus class:
 
-`$ python3 ../modbus/tools/generate-connection.py -j registers.json -o . -c MyModbusConnection`
+`$ python3 ../modbus/tools/generate-connection.py -j registers.json -o . -c MyModbus`
 
 You the result will be a header and a source file called:
```

Comment:
-You

Reply:
Updated section: 70dc99a33f1ae34dd27815bf37d87a21507af28b